src/components/JobSectionList/JobCard/index.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import { device } from 'utils';
+
+const JobCard = ({ data }) => {
+  const { jobTitle, jobURL, jobShortDesc } = data;
+  return (
+    <JobCardContainer>
+      <JobCardHeader>
+        <JobTitle>{jobTitle}</JobTitle>
+        <JobLink href={jobURL}>Learn more & apply</JobLink>
+      </JobCardHeader>
+      <JobCardDesc>{jobShortDesc}</JobCardDesc>
+    </JobCardContainer>
+  );
+};
+
+export default JobCard;
+
+JobCard.propTypes = {
+  data: PropTypes.object,
+};
```

Comment:
be more explicit with this

```suggestion
JobCard.propTypes = {
  data: PropTypes.shape({
    jobTitle: PropTypes.string,
    jobURL: PropTypes.string,
    jobShortDesc: PropTypes.string,
  })
};
```